tools/start-db-services

```diff
@@ -29,7 +29,7 @@ if [[ -f /etc/default/bios-db-rw ]]; then
     exit 0
 fi
 
-sudo /bin/systemctl start fty-db-init || die "Starting fty-db-init failed"
+sudo /bin/systemctl start fty-db-firstboot || die "Starting fty-db-firstboot failed"
```

Comment:
I assume your testing proved that a not-enabled service can be started this way?